game-guide/trading.md

```diff
@@ -1,54 +1,297 @@
-4636 - Trade Master 4536 - Planetary Super Freighter 4276 - Inter-Stellar Trader 3830 - Planetary Freighter 3715 - Thief
+There are a few limits in code that have been stripped out of the equations as they are.  The stock amount over stock max fraction will be between 0 and 1.  Relations are capped at 1000.  The rest variables are limited by game behavior but not explicitly hard coded.
 
-All of the above are available to all traders except two ships, the Trade Master, which you must be Alskant to purchase, and the Thief, in which you must be evil to purchase. The numbers next to the ships are the amount of exp that ship would make per day based on trading a perfect 3x route located 3 sectors from where you park your ship.
+It is possible, by using the equations above, to calculate the perfect value for each transaction.  It is an incredibly effective method to increase experience and profit earlier in the game.  It is also dangerously distracting to sit in a defenseless trade ship while keying in supply and relation numbers for every trade.  Experiment at your own risk.
 
-These figures (and therefore the rankings) of the ships change a bit when you take into account some more variations on route length and distance from where you park, and distance to the bank.
+## Experience Mechanics
 
-Basically, the more non trading moves you need to make each day, the better the faster ships become. Non trading moves include moving to and from fed space, or between routes, or to visit the bank. For example, if you make 20 non trading moves each day, and trade on a perfect 3* route, the PSF drops down the rankings to 4th best, behind the TM, the IST and the Thief.
+Experience Gain = (Cargo Holds / 13) &middot; Distance Factor
 
-On a strictly trading basis, if you are making more than 16 non trading moves a day the IST is ahead of the PSF for experience earned. Of course the PSF is much more defendable, but the IST can trade anywhere in the universe quite easily, so is much less predictable.
+Moving goods with a greater distance factor is a huge part of gaining experience.  The person who moves the most goods along the longest route will usually end up with the most.
 
-One final note: In order to gain Federal protection at the end of your trading session, you CANNOT have any illegal goods on board. So make sure that you have all of them sold off before heading to Fed, and make sure that you have sold off your weapons to get your attack rating down. (See Alignment for more detail on attack ratings and Federal Protection.)
+Experience is also tricky because of the bartering process.  There is an ideal price that must be guessed quite accurately in order to get full experience.  But the port will offer you a price that, if your relations are not at least 1000, will not be optimal.  Guessing the correct value is incredibly difficult due to the number of variables used in the process.  Most people opt simply to work their relations up as fast as possible.
```

Comment:
```suggestion
Experience is also tricky because of the bartering process.  There is an ideal price that must be guessed quite accurately in order to get full experience.  However, if your relations are 1000 or above, then the port will give you the best price without needing to barter.  Guessing the correct value is difficult due to the number of variables used in the calculation.  Most people opt simply to work their relations up as fast as possible.
```